tools/code-owners-parser/CodeOwnersManualTester/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Azure.Sdk.Tools.CodeOwnersParser;
+
+namespace CodeOwnersManualTester
+{
+    internal class Program
```

Comment:
Do we need this tool in addition to [CodeownersManualAnalysisTests](https://github.com/Azure/azure-sdk-tools/blob/c37701ec43d68f96f480ddf7fc502841be7fbaed/tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners.Tests/CodeownersManualAnalysisTests.cs#L40) ?

Reply:
I'm removing this project with the next commit